modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -190,7 +190,8 @@ public List<JSONObject> getVerticalNavigationJSONObjects()
 			JSONArray navigationItemsJSONArray = null;
 
 			if (!mvcPath.equals("/admin/view_kb_suggestions.jsp") &&
-				!mvcPath.equals("/admin/view_kb_templates.jsp")) {
+				!mvcPath.equals("/admin/view_kb_templates.jsp") &&
+				!mvcPath.equals("/admin/view_kb_template.jsp")) {
```

Comment:
do we really want to mantain both paths? Seems more like a mistake... @adolfopa @AliciaGarciaGarcia ?